test/sol5/kyberNetworkProxy.js

```diff
@@ -578,6 +578,41 @@ contract('KyberNetworkProxy', function(accounts) {
             });
         }
     });
+    
+    describe("test enable", async () => {
+        it("test set enable with admin", async () => {
+            await network.setEnable(false, { from: admin });
+            let enable = await networkProxy.enabled();
+            assert(!enable);
+            await network.setEnable(true, { from: admin });
+            enable = await networkProxy.enabled();
+            assert(enable);
+        });
+    });
+
+    describe("test swap function (backward compatible)", async () => {
```

Comment:
please rename this to:

test simple swap API
although its backward compatible.
its not the reason we have them, we have them for having some simple swap API